routes/entryRoutes.js

```diff
@@ -115,4 +118,31 @@ routes.GETstudentEntries = function(req, res){
   });
 }
 
+routes.GETcohortEntries = function(req, res) {
+  var cohort = req.params.cohort;
+  var studentListFull = [];
+  var attendanceList = [];
+  var starsList = [];
+  var datesList = [];
+  var behaviorList = [];
+  var warningList = [];
+  var engageContentList = []
+
+  Student.find({program:cohort}, function(err, students){
+    var studentIds = students.map(function(student){return student._id});
+    console.log('studentIds: '+studentIds);
+    FormDB.find({_studentID:{$in:studentIds}}, function(err, forms){
+      forms.reduce(function(prev,form){
```

Comment:
can you explain this? I tried looking at documentation and didn't entirely get it. If you understand and could explain, that'd be awesome! otherwise, I'll look further into it tonight and try to figure it out.